kratos/testing_/test_skipped_exception.h

```diff
@@ -0,0 +1,129 @@
+//    |  /           |
```

Comment:
`testing_` folder?

Reply:
I see many renamings because of this

Reply:
This is leftover, I will remove it, don't worry. Its there because I still need to use things from the older testing.